core/domain/prod_validators.py

```diff
@@ -4133,7 +4133,7 @@ class UserSubscriptionsModelValidator(
     def _get_external_id_relationships(cls, item):
         return [
             base_model_validators.ExternalModelFetcherDetails(
-                'activity_ids', exp_models.ExplorationModel, item.activity_ids),
+                'entity_ids', exp_models.ExplorationModel, item.activity_ids),
```

Comment:
Subscriptions models should still refer to "activity IDs", if activities relates to explorations + collections only. Could you please fix this throughout?

Reply:
Actually subscriptions have both `activity_ids` (ought to be renamed to `exploration_ids`, I have added an issue) and `collection_ids`.